functional/basic-attestation-without-mtls/test.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+        limeBackupConfig
+        # update /etc/keylime.conf
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        rlRun "limeUpdateConf cloud_agent mtls_cert_enabled False"
+        rlRun "limeUpdateConf cloud_agent enable_insecure_payload True"
+        rlRun "limeUpdateConf cloud_verifier agent_mtls_cert_enabled False"
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            # make sure tpm2-abrmd is running
+            rlServiceStart tpm2-abrmd
+            sleep 5
+            # start ima emulator
+            export TPM2TOOLS_TCTI=tabrmd:bus_name=com.intel.tss2.Tabrmd
+            # workaround for https://github.com/keylime/rust-keylime/pull/286
+            export PATH=/usr/bin:$PATH
```

Comment:
Please remove this workaround, I believe it is not needed anymore.

Reply:
Will do.